src/blocks/PerilsBlock/PerilsBlock.tsx

```diff
@@ -33,22 +35,24 @@ export const PerilsBlock: React.FC<PerilsBlockProps> = ({
   index,
   size,
   insurance_types,
+  compare_perils,
 }) => {
   const { currentLocale } = useLocale()
   const insuranceTypes = useMemo(
     () => insurance_types.map((insuranceType) => insuranceType.value),
     [insurance_types],
   )
 
-  const perils = usePerils(insuranceTypes, currentLocale.iso)
+  let perils = usePerils(insuranceTypes, currentLocale.iso)
+  perils = compare_perils ? getPerilsComparison(perils) : perils
```

Comment:
Should this perhaps be in a `useMemo`? Otherwise these will be re-computed ever render, right?

Reply:
That's probably a good idea! I'm a bit bad at distinguishing when to use them :)